less/components/sidebar.less

```diff
@@ -19,7 +19,7 @@ html.no-js .sidebar {
   &.teach-like-mozilla .sidebar {
     .sidebarColorizer(@tealSidebarColor);
   }
-  &.clubs .sidebar {
+  &.clubs .sidebar, &.clubs-curriculum-page .sidebar {
```

Comment:
Direction @vazquez ? should I just do `darken(@orangeSidebarColor, 10%)` or something similar?